nixos/wiki.html

```diff
@@ -0,0 +1,31 @@
+<h1>The NixOS wiki is currently down.</h1>
```

Comment:
"is currently down" implies a temporary issue, like a server problem. That's likely to cause people to constantly ping us about this. Maybe "The NixOS wiki has been shut down.".